portfolio/src/main/webapp/index.html

```diff
@@ -18,32 +18,14 @@ <h1>Anthony's Super Amazing Portfolio</h1>
         friends and family. My favorite artist is J. Cole and Kendrick Lamar is
         second.
       </p>
-      <p>Click the button to see some pictures of me! (5 total)</p>
-      <button
-        type="button"
-        id="picture-and-caption-button"
-        onclick="getPictureAndCaption()"
-      >
-        Hello
-      </button>
       <div id="picture-and-caption">
         <div id="picture"></div>
+        <button type="button" id="previous">&#10094;</button>
+        <button type="button" id="next">&#10095;</button>
```

Comment:
nit: "previous" and "next" are a little cryptic. Maybe "next-picture-button" or similar would be better.

The main reason for this is that it would clarify your code in script.js: 

> const nextButton = document.getElementById('next-picture-button');

Reference: https://drive.google.com/file/d/1uzaIjN8oLqdVL6Sp0VE7ZnBoehEfg_rz/view